web-components/src/components/code-editor/CodeEditor.ts

```diff
@@ -0,0 +1,227 @@
+import reset from "@/wc_scss/reset.scss";
+import {
+  customElement,
+  html,
+  LitElement,
+  query,
+  property,
+  PropertyValues,
+  internalProperty,
+  queryAssignedNodes
+} from "lit-element";
+import styles from "./scss/module.scss";
+import hljs from "highlight.js/lib/core";
+import "@/components/button/Button";
+
+import { ifDefined } from "lit-html/directives/if-defined";
+import { nothing } from "lit-html";
+@customElement("md-code-editor")
+export class CodeEditor extends LitElement {
+  @property({ type: String, attribute: "accept-language" }) acceptLanguage = "javascript";
+  @property({ type: String }) getLocalization = "Get";
+  @property({ type: String }) copyLocalization = "Copy";
+  @property({ type: String }) copiedLocalization = "Copied";
+
+  @query("input[type='file']") input!: HTMLInputElement;
+  @query(".md-code-editor-code-block") codeBlock!: HTMLPreElement;
+
+  @internalProperty() private disableCopyButton = true;
+  @internalProperty() private acceptTypes = "";
+  @internalProperty() private fileName = "";
+  @internalProperty() private copied = false;
+
+  @queryAssignedNodes("code-block") slotNodes!: Node[];
+
+  static get styles() {
+    return [reset, styles];
+  }
+
+  triggerFileLoad(event: MouseEvent) {
```

Comment:
Get is not a button - GET is a type of API request/response for API reference, It simply should be a badge with variation for GET/POST